universal-login-relayer/src/core/utils/errors.ts

```diff
@@ -154,6 +178,13 @@ export class NotFound extends RelayerError {
   }
 }
 
+export class EmailNotFound extends NotFound {
+  constructor(email: string) {
+    super(`Email confirmation is not requested for ${email}`, 'EmailNotFound');
```

Comment:
`Email confirmation not found for email: ${email}`

Reply:
I think `Email confirmation is not requested for` is more descriptive

Reply:
Your version says that confirmation is optional